crawler/containers.py

```diff
@@ -47,16 +47,15 @@ def get_filtered_list_of_containers(
     The partitioning is given by `partition_strategy`.
     """
 
-    environment = options.get('environment', defaults.DEFAULT_ENVIRONMENT)
+    environment = options.get('environment',
+                              get_config()['general']['environment'])
```

Comment:
[Tag:R1] this is confusing to me- is this an artifact of supporting both CLI and config file right now? Because options.get() seems to me to be giving the same meaning as get_config(), but from what I understand you are using the latter to get default value of 'environment'.